CHANGELOG.md

```diff
@@ -18,6 +18,11 @@
   names to start with an underscore.
   [JP Simard](https://github.com/jpsim)
 
+* Disable and re-enable rules from within source code comments using
+  `// swiftlint:disable:$IDENTIFIER` and `// swiftlint:enable:$IDENTIFIER`.  
```

Comment:
that *would* make it more similar to rubocop, and I have no main reason to keep that separator a colon... so ok!